$OpenBSD$

main.c:376: warning: implicit declaration of function '_thread_sys_closefrom'
(XXX rework after rthreads are enabled)

--- scheduler/main.c.orig	Mon Feb  6 14:02:24 2012
+++ scheduler/main.c	Mon Feb  6 14:03:01 2012
@@ -69,6 +69,9 @@
 #  include <notify.h>
 #endif /* HAVE_NOTIFY_H */
 
+#ifdef __OpenBSD__
+int	_thread_sys_closefrom(int);
+#endif
 
 /*
  * Local functions...
